setup.py

```diff
@@ -43,10 +43,13 @@
         ]
     },
     zip_safe=False,
+    dependency_links=(
+        "git+https://github.com/genialis/resolwe.git@master#egg=resolwe-master",
+    ),
     install_requires=(
         'Django~=1.10.5',
         'djangorestframework>=3.4.0',
-        'resolwe~=1.4.1',
```

Comment:
I recently discovered that using `resolwe==master` brakes some Python tools. Please, use `resolwe==1.4.1` and update the dependency link to:
`"git+https://github.com/genialis/resolwe.git@master#egg=resolwe-1.4.1"`.